contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol

```diff
@@ -479,11 +484,10 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
     bytes memory offchainConfig
   ) internal {
     if (s_hotVars.paused) revert RegistryPaused();
-    if (!upkeep.target.isContract()) revert NotAContract();
     if (checkData.length > s_storage.maxCheckDataSize) revert CheckDataExceedsLimit();
     if (upkeep.performGas < PERFORM_GAS_MIN || upkeep.performGas > s_storage.maxPerformGas)
       revert GasLimitOutsideRange();
-    if (s_upkeep[id].target != ZERO_ADDRESS) revert UpkeepAlreadyExists();
+    if (s_upkeep[id].performGas != 0) revert UpkeepAlreadyExists();
```

Comment:
can we check the forwarder address here? (since that is sort of the pseudo target address)